xclim/data/fr.json

```
@@ -846,12 +846,24 @@
     "title": "Approximation de la neige",
     "abstract": "Précipitation solide estimée à partir de la précipitation totale et de la température selon une méthode et un seuil de température donnés."
   },
+  "SNOW_AMOUNT_APPROXIMATION": {
+    "long_name": "Approximation de la quantité de neige quotidienne à partir de l'épaisseur et de la densité de la neige",
```

```suggestion
    "long_name": "Quantité de neige à la surface",
```
I think the long name should refer to the content of the variable and not to it's source, which is more the role of `description`.